rootfs/etc/profile.d/_20-mounts.sh

```
@@ -0,0 +1,206 @@
+		local src="/.FS_HOST"
+		local dest="/.FS_CONT"
+		if [[ -d "${src}" ]]; then
+			map="${dest}"
+			mkdir -p "${dest}"
+		else
+			red "# ERROR: Supposed host mount directory ${src} does not exist. Fatal error."
+			return 9
+		fi
+		GEODESIC_HOST_PATHS+=("${src}/" "${dest}/")
+		if [[ -z $GEODESIC_HOST_UID ]] || [[ -z $GEODESIC_HOST_GID ]]; then
+			red '# ERROR: `$MAP_FILE_OWNERSHIP` is set to "true" but `$GEODESIC_HOST_UID` and `$GEODESIC_HOST_GID` are not set.'
```

Shellcheck (0.10.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)